src/main/java/com/codeborne/selenide/webdriver/HttpClientTimeouts.java

```diff
@@ -13,5 +13,5 @@
  */
 @ParametersAreNonnullByDefault
 public class HttpClientTimeouts {
```

Comment:
It seems will be better get rid of this at all and use default from global config.

Reply:
We cannot do it because we still need to use `HttpClientTimeouts.defaultReadTimeout` when running browser locally (in class `SelenideNettyClientFactory`). 

But yes, we could move field `defaultReadTimeout` from `HttpClientTimeouts` into `SelenideNettyClientFactory` and remove class `HttpClientTimeouts`.